docs/content/operations-as-service/offer.md

```diff
@@ -0,0 +1,5 @@
+`Version: 6 Nov 2022`
+
+# Operations and Maintenance as a Service
+
+In addition to the `Stakater App Agility Platform`, Stakater can provide operational and maintenance support for existing production clusters for flexible time periods. The Stakater Global SRE Team will be responsible for the operational efforts.
```

Comment:
@karl-johan-grahn maybe we should remove the word production; as customer might want us to manage their non-prod clusters as well